packages/postgres-database/src/simcore_postgres_database/migration/versions/99f9b200801c_adde_sharing_networks_table.py

```diff
@@ -0,0 +1,33 @@
+"""adde sharing networks table
+
+Revision ID: 99f9b200801c
+Revises: a6b281f3ee15
+Create Date: 2022-03-07 08:54:31.894295+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '99f9b200801c'
+down_revision = 'a6b281f3ee15'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('sharing_networks',
```

Comment:
what about ```projects_networks```? this way we have all the tables related to projects together?

Reply:
yes, this was renamed.